tests/factories.py

```diff
@@ -1,63 +1,44 @@
-from dataclasses import dataclass
-
 from django.conf import settings
 
 from airlock.users import User
 from airlock.workspace_api import ReleaseRequest, Workspace, generate_request_id
 
 
-default_user = User(1, "test", [], True)
-
-
-class FileWriterMixin:
-    def path(self):  # pragma: no cover
-        return NotImplemented()
-
-    def mkdir(self, path):
-        fp = self.path() / path
-        fp.mkdir(parents=True, exist_ok=True)
-
-    def write_file(self, path, contents=""):
-        fp = self.path() / path
-        self.mkdir(fp.parent)
-        fp.write_text(contents)
-
+default_user = User(1, "testuser")
 
-@dataclass
-class WorkspaceFactory(FileWriterMixin):
-    name: str
 
-    def __post_init__(self):
-        self.path().mkdir(parents=True, exist_ok=True)
+def ensure_workspace(workspace_or_name):
+    if isinstance(workspace_or_name, str):
+        return create_workspace(workspace_or_name)
+    elif isinstance(workspace_or_name, Workspace):
+        return workspace_or_name
 
-    def path(self):
-        return settings.WORKSPACE_DIR / self.name
+    raise Exception(f"Invalid workspace: {workspace_or_name})")  # pragma: nocover
 
-    def create_request(self, request_id):
-        return ReleaseRequestFactory(request_id, self.name)
 
-    def create_request_for_user(self, user=None):
-        if user is None:
-            user = default_user  # pragma: nocover
-        request_id = generate_request_id(self.name, user)
-        return ReleaseRequestFactory(request_id, self.name)
+def create_workspace(name):
+    workspace_dir = settings.WORKSPACE_DIR / name
+    workspace_dir.mkdir(exist_ok=True, parents=True)
+    return Workspace(name)
 
-    def get(self):
-        return Workspace(self.name)
 
+def create_request(workspace, user=default_user, request_id=None):
```

Comment:
Can we name this `create_release_request`? I know it's more verbose, but `request` is so easily confused with http request and I had to check back to be sure of which one it was